src/main/java/com/kadirakinkorkunc/urlshortener/mapper/ConfigurationMapper.java

```diff
@@ -0,0 +1,60 @@
+package com.kadirakinkorkunc.urlshortener.mapper;
+
+import com.kadirakinkorkunc.urlshortener.model.Configuration;
+import com.kadirakinkorkunc.urlshortener.model.ConfigurationRequest;
+import com.kadirakinkorkunc.urlshortener.model.ConfigurationResponse;
+import com.kadirakinkorkunc.urlshortener.util.ConfigurationUtils;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ConfigurationMapper {
+
+    public ConfigurationResponse mapConfigurationToConfigurationResponse(Configuration configuration) {
+        ConfigurationResponse configurationResponse = new ConfigurationResponse();
+        configurationResponse.setKey(configuration.getKey());
+        configurationResponse.setValue(configuration.getValue());
+        return configurationResponse;
+    }
+
+    public List<ConfigurationResponse> mapConfigurationsToConfigurationResponses(List<Configuration> configurationList) {
+        return configurationList.stream().map(this::mapConfigurationToConfigurationResponse).collect(Collectors.toList());
+    }
+
+    public Configuration mapConfigurationReqToConfiguration(ConfigurationRequest configurationRequest) {
```

Comment:
I don't like method names to contain abbreviations.. Instead of this name, I think you can change the name to:
```suggestion
    public Configuration mapConfigurationRequestToConfiguration(ConfigurationRequest configurationRequest) {
```